### PR TITLE
PMM-13556 Add watchtower submodule.

### PR DESCRIPTION
**Pull request by talhabinrizwan**
https://jira.percona.com/browse/PMM-13556

### CODE REVIEW

File: .gitmodules
Comment (ademidoff) [inline .gitmodules:54]: I think we need to add it to `ci-default.yml` as well, at least until we merge [this PR](https://github.com/Percona-Lab/pmm-submodules/pull/3774).

**Review by BupycHuk — APPROVED** [review]
please don't merge it until Beta release

**Comment by JNKPercona** [comment]
Server docker: perconalab/pmm-server-fb:PR-3777-b3e94ec
Client docker: perconalab/pmm-client-fb:PR-3777-b3e94ec
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3777-b3e94ec.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3777-b3e94ec&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3777-b3e94ec.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/1516/

**Comment by JNKPercona** [comment]
Server docker: perconalab/pmm-server-fb:PR-3777-911013f
Client docker: perconalab/pmm-client-fb:PR-3777-911013f
Client tarball: https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3777-911013f.tar.gz
Staging instance: https://pmm.cd.percona.com/job/pmm3-aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3777-911013f&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm-client/pmm-client-PR-3777-911013f.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed: https://pmm.cd.percona.com/job/pmm3-api-tests/1574/

**Pull request merged**
